lib/saluki-components/src/transforms/host_enrichment/mod.rs

```diff
@@ -73,6 +73,8 @@ impl SynchronousTransform for HostEnrichment {
         for event in event_buffer {
             match event {
                 Event::Metric(metric) => self.enrich_metric(metric),
+                Event::EventD(_) => todo!(),
```

Comment:
Similar comment to the one above for the DogStatsD source around a mutable fallible conversion function (i.e. `Event::as_metric_mut`).